src/componentes/CriarSenha/CriarSenha.jsx

```
@@ -1,70 +1,87 @@
+	button,
+	setEtapa,
+	submitCadastro,
+	submitCadastroIP,
+	arg,
+	data,
+}) => {
+	const [senha, setSenha] = useState("");
+	const [confirmarSenha, setConfirmarSenha] = useState("");
+	const [message, setMessage] = useState("");
+	const match = (e) => {
+		if (senha == e || e == confirmarSenha) {
```

Biome (1.9.4)

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)